src/styles/blocks/card-style.scss

```diff
@@ -0,0 +1,137 @@
+
+
+* {
+  margin: 0;
+}
+
+body {
+  margin: 0;
+  font-family: $primary-font;
+  font-weight: 400;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.container {
+  display: flex;
+  gap: 50px;
+}
+
+.card {
```

Comment:
In this file should be styles only for card block. Move all other styles to index.scss file.
Also, would be great to name the file the same as block (card.scss)